style.css

```diff
@@ -0,0 +1,271 @@
+*{
+    margin:0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+:root{
+    --primary-color:#424874;
+    --secondary-color:#a6b1e1;
+    --txt-primary:#dcd6f7;
+    --txt-secondary:#d1d4e4;
+    --txt-dark:#424874;
+    --txt-light:#a6b1e1;
+    --padding-sm:0.5rem;
+    --padding-md:1rem;
+    --padding-lg:1.5rem;
+}
+html {
+    scroll-behavior: smooth;
+    overflow-x: hidden;
+}
+body{
+    width: 100vw;
+    min-height: 100vh;
+    background: #f7f7f7;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    overflow-x: hidden;
+}
+a{
+    text-decoration: none;
+    color: inherit;
+}
+p{
+    line-height: 1.8rem;
+}
+h1{
+    margin:1rem 0;
+}
+.header-index{
+    width: 100vw;
+    padding: 1rem 1rem;
+    position: sticky;
+    top:0;
+    z-index: 2;
+    background: #f7f7f7;
+}
+.header{
+    width: 100vw;
+    padding: 1rem 1rem;
+    position: sticky;
+    top:0;
+    z-index: 2;
+    background: #f7f7f7;
+    display: none;
+}
+.brand-link{
+    color:var(--primary-color);
+    font-size: 1.2rem;
+}
+.brand-name{
+    display: inline-flex;
+    align-items: center;
+    width:max-content;
+    cursor:pointer;
+}
+.brand-logo{
+    width:3rem;
+    margin-right: 0.5rem;
+}
+main{
+    margin: auto 0;
+    width: 75%;
+    max-width: 30rem;
+    text-align: center;
+}
+main > .brand-logo{
+    width: 8rem;
+}
+.brand-tagline{
+    font-size: 1.5rem;
+    color: var(--primary-color);
+    margin:1rem 0;
+    letter-spacing: 0.1rem;
+}
+.btn{
+    padding-top: 0.8rem !important;
+    padding-bottom: 0.8rem !important;
+}
+.main{
+    width: 100vw;
+}
+.side-bar-navigation{
+    position: fixed;
+    min-width: max-content;
+    height: 100vh;
+    top: 0;
+    left:0;
+}
+.side-bar{
+    display: flex;
+    flex-direction: column;
+    background: #f7f7f7;
+    height: max-content;
+    height: 100%;
+    padding: var(--padding-md);
+    padding-right: 0;
+    z-index: 2;
+}
+#brand{
+    margin-bottom: 1rem;
+}
+.heading-md:hover{
+    color: var(--txt-light);
+}
+.component-list__container{
+    display: flex;
+    flex-direction: column;
+}
+
+.component-item{
+    color: var(--txt-dark);
+    padding:var(--padding-sm);
+    border-radius: 0.5rem;
+    width: max-content;
+    cursor: pointer;
+    list-style: none;
+    transition: 0.2s;
+}
+
+.component-item:hover{
+    transform: scale(1.1);
+    background: var(--secondary-color);
+    color: var(--txt-primary);
+}
+.docs__container{
```

Comment:
Hey, 
All your class names are following the `class-name` or `class-name__subname` pattern, but here your `docs__container` class is following a different naming convention. So to maintain consistency, you can rename it.

Reply:
I know all my class names in the component css follows same pattern, Some classes in css of my documentation does not.
My component is located in css/component.css